vispy/scene/canvas.py

```diff
@@ -87,17 +87,20 @@ def __init__(self, *args, **kwargs):
         self._fb_stack = []  # for storing information about framebuffers used
         self._vp_stack = []  # for storing information about viewports used
         self._scene = None
-        self._bgcolor = Color(kwargs.pop('bgcolor', 'black')).rgba
```

Comment:
actually, it *could* potentially cause a problem if something later (like a draw handler) gets called by the superclass init and expects `self._bgcolor` to already exist... so in this case I would say go ahead and violate DRY and just have the `Color` interpreting done in two places -- here and the setter.